lib/topological_inventory/providers/common/operations/source.rb

```diff
@@ -211,7 +211,8 @@ def availability_status_message(payload)
             messaging_client.publish_topic(
               :service => SERVICE_NAME,
               :event   => EVENT_AVAILABILITY_STATUS,
-              :payload => payload.to_json
+              :payload => payload.to_json,
+              :headers => identity
```

Comment:
This makes sense and is the way to fix this, but to verify, are other services adding the identity header to kafka topic? I only ask because this came up early on in catalog because kafka is global to the entire platform.

Reply:
Yep, it looks like catalog/approval share the x-rh-identity and x-rh-request-id in the same way: 

https://github.com/lindgrenj6/catalog-api/blob/3e3aeb97f3258993c509c224a8e8d359a152787b/lib/kafka_listener.rb#L34-L40